ci/jenkins/README.md

```diff
@@ -1 +1,38 @@
 # Nephe CI: Jenkins
+
+This directory includes all the scripts required to run CI for nephe.
+
+We have static VM as jenkins node, when there is a pull request or daily build trigger the job. The code will be cloned on
+static VM, and it will call terraform to deploy dynamic VM and use ansible for system configuration such as installing docker,
+and then run tests on the dynamic VM, destroy the VM once the test succeeds or fails.
+
+## Terraform
+We create terrform tfvars file in nephe-ci.sh with jenkins credetials. `deploy.sh` is the script to create vm, and store the
```

Comment:
Please add hyperlinks when you are referring to scripts.
For example: [deploy.sh](deploy.sh)

Reply:
done